model/lifecycle_test.go

```diff
@@ -610,21 +615,22 @@ func TestBuildSetActivated(t *testing.T) {
 					Status:       evergreen.TaskUndispatched,
 					Activated:    false,
 					DispatchTime: utility.ZeroTime,
+					ActivatedBy:  evergreen.GenerateTasksActivator,
 				}
 				So(canary.Insert(), ShouldBeNil)
 
-				So(ActivateBuildsAndTasks(ctx, []string{b.Id}, false, ""), ShouldBeNil)
+				So(ActivateBuildsAndTasks(ctx, []string{b.Id}, false, evergreen.GenerateTasksActivator), ShouldBeNil)
 				// the build should have been updated in the db
 				b, err := build.FindOne(build.ById(b.Id))
 				So(err, ShouldBeNil)
 				So(b.Activated, ShouldBeFalse)
-				So(b.ActivatedBy, ShouldEqual, "")
+				So(b.ActivatedBy, ShouldEqual, evergreen.GenerateTasksActivator)
 
 				// only the matching task should have been updated that has not been set by a user
 				deactivatedTasks, err := task.Find(task.ByActivation(false))
 				So(err, ShouldBeNil)
 				So(len(deactivatedTasks), ShouldEqual, 3)
-				So(deactivatedTasks[0].Id, ShouldEqual, matching.Id)
+				So(deactivatedTasks[0].Id, ShouldEqual, matching.Id) // So(deactivatedTasks[0].Id, ShouldEqual, differentUser.Id)
```

Comment:
Was the comment an accident? :0 
```suggestion
				So(deactivatedTasks[0].Id, ShouldEqual, matching.Id)
```